src/utils/azureUtils.ts

```diff
@@ -65,6 +76,18 @@ export function getIconPath(type?: string, kind?: string): TreeItemIconPath {
     return treeUtils.getIconPath(iconName);
 }
 
+export async function getArmTagKeys(context: IActionContext): Promise<Set<string>> {
+    const armTagKeys: Set<string> = new Set();
+    for (const sub of (await ext.rootAccountTreeItem.getCachedChildren(context))) {
```

Comment:
This can take a while if the user has multiple subscriptions opened, but not sure what we can do about that.

Reply:
Yeah I think it's unavoidable

Reply:
I don't know if this is even possible with an input/quickpick/whichever, but could we async load the keys, and while they're waiting let them paste or type something if they already know what they want?